src/v2/Components/Artwork/FillwidthItem.tsx

```diff
@@ -177,6 +178,7 @@ export const FillwidthItem = styled(FillwidthItemContainer).attrs<
   width: ${props => props.width}px;
   vertical-align: top;
   margin-right: ${props => props.margin || 0}px;
+  margin-left: ${props => props.leftMargin || 0}px;
```

Comment:
This is starting to get a bit wiley with the `margin` prop just above. Possible to audit use of `margin` in codebase and replace with `marginRight` to match this prop addition?